packages/app-generic-asset/src/Available.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017-2020 @polkadot/react-query authors & contributors
```

Comment:
Should we mention Centrality here?

Reply:
I think it's ok, we actually fall under "contributors"